data/categories.txt

```diff
@@ -3666,6 +3684,8 @@ fa:اداره پست
 
 amenity-waste_disposal
 en:Dumpster|Trash|bin|garbage|waste disposal|U+1F6AE
+en-GB:Waste disposal|Trash|bin|garbage|U+1F6AE
```

Comment:
I'd add prefix 4 here and below for the hint.

Reply:
Sure.